scripts/enforce-change-notes-updated-on-pr-to-master

```diff
@@ -5,12 +5,7 @@ set -e -o pipefail -u
 self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
-change_notes_path=CHANGE-NOTES.html
-
-function change_notes_from_git_revision() {
-  local revision=$1
-  git show "$revision:$change_notes_path" 2>/dev/null || true
-}
+change_notes_path=CHANGE-NOTES.md
```

Comment:
Also! pls note down the guidelines for updating changelog (esp. about `patchChangelog` task) in CONTRIBUTING.md, in a relevant section